cosmoz-image-viewer.js

```diff
@@ -403,11 +395,22 @@
 			this.hidden = value;
 		},
 
-		_imageListChanged(newlist) {
-			if (!newlist) {
-				return;
-			}
+		_imageListChanged(images) {
 			this.currentImageIndex = 0;
+			/**
+			 * _resolvedImages = []
+			 * Solves: when changing the images property to a new array of images,
+			 * the displayed images might not be updated if the new array has the same size as the previous one.
+			 * See https://github.com/Neovici/cosmoz-image-viewer/issues/21
+			 */
+			this.set('_resolvedImages', []);
+			setTimeout(() => {
```

Comment:
`dom-repeat` _debounce_ rendering using 
```
Polymer.dom.addDebouncer(this.debounce(...))
```

so it will be safer if you use something similar (to be sure the empty array rendering will be done by dom-repeat)